index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="sass/main.css">
+    <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <link rel="stylesheet" href="./webfontkit-20220518-164003/myfont.css">
+    <title>Book-Store</title>
+</head>
+
+<body>
+
+    <header>
+        <div class="pre-navbar bg-color">
+            <ul>
+                <li><img src="./img/Social1.png" alt=""></li>
+                <li><img src="./img/Social2.png" alt=""></li>
+                <li>SignIn</li>
+                <li>My Page</li>
+                <li>Language</li>
+            </ul>
+        </div>
+        <nav class="nav-bar">
+            <!-- Mobile-nav -->
+            <div class="mobile-nav">
+                <input id="menu-toggle" type="checkbox" />
+                <label class='menu-button menu-button-container' id="menuButton" for="menu-toggle">
+                </label>
+
+                <ul class="menu">
+                    <li><a class="menu-item" href="./About.html">About</a></li>
+                    <li><a class="menu-item" href="#">BuyNow</a></li>
+                    <li><a class="menu-item" href="#">Contact</a></li>
+                </ul>
+            </div>
+            <!-- End Mobile Nav -->
+
+            <!-- Start Desktop Nav -->
+            <div class="desktopnav">
+                <a href="./index.html"><img class="navbar-logo" src="./img/logo.png" alt="" width=130></a>
+                <ul class="Desktop-Menu">
+                    <li><a class="Desktopmenu-item" href="./About.html">About</a></li>
+                    <li><a class="Desktopmenu-item" href="">BuyNow</a></li>
+                    <li><a class="Desktopmenu-item" href="">Contact</a></li>
+                    <li>
+                        <span id="unique-item">BookStore</span>
+                    </li>
+                </ul>
+            </div>
+            <!-- End Desktop -->
+
+        </nav>
+    </header>
+
+
+
```

Comment:
- [ ] Kindly remove all unnecessary blank space from the code to follow best practices as this will make your code clean and readable.